common/static/coffee/src/discussion/utils.coffee

```diff
@@ -113,7 +113,7 @@ class @DiscussionUtil
       alertDiv = $("<div class='modal' role='alertdialog' id='discussion-alert' aria-describedby='discussion-alert-message'/>").css("display", "none")
       alertDiv.html(
         "<div class='inner-wrapper discussion-alert-wrapper'>" +
-        "  <button class='close-modal dismiss' title='Close modal'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
+        "  <button class='close-modal dismiss' title='" + gettext("Close") + "'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
```

Comment:
Did you verify that this works (the generated HTML looks correct)? I'm not sure if you've been updating the sandbox for changes, so I'm not testing manually myself.